src/edu/csus/ecs/pc2/services/eventFeed/WebServer.java

```diff
@@ -300,6 +297,28 @@ private Logger getLog() {
     private SecurityHandler basicAuth() {
 
         HashLoginService l = new HashLoginService();
+        
+        // First, load PC2 accounts into jetty - this will allow PC2 users to use the API with their login id
+        //    Only teams, admins and judges
+        Account[] accounts = contest.getAccounts();
+        String role;
+        Type clientType;
+        
+        for (Account account: accounts) {
+            clientType = account.getClientId().getClientType();
+            if (clientType == Type.TEAM) {
+                role = WEBAPI_ROLE_TEAM;
+            } else if (clientType == Type.ADMINISTRATOR) {
+                role = WEBAPI_ROLE_ADMIN;
+            } else if (clientType == Type.JUDGE) {
+                role = WEBAPI_ROLE_JUDGE;
+            } else {
+                continue;
+            }
```

Comment:
nothing for analysts here?

Reply:
That piece of code add's accounts from PC2 to the `HashLoginService `for Web Server **SecurityHandler**.  There are no analyst accounts in PC2 (that I know of).   Analysts, and icpclive accounts will have to be added to the **realm.properties** file by hand, unless we add new account types to PC2.